python/tvm/contrib/hexagon/session.py

```diff
@@ -286,6 +294,11 @@ def _aot_executor_from_factory(
             for target in module.target.values()
```

Comment:
nit: Need some clean up [here](https://github.com/apache/tvm/pull/11065/files#diff-af4d2019fb7bd2f88bd9209309d4008514cdf340047d3019b41e8f4fbcbbdac0R281) and [here](https://github.com/apache/tvm/pull/11065/files#diff-af4d2019fb7bd2f88bd9209309d4008514cdf340047d3019b41e8f4fbcbbdac0R287), to reference AOT not graph.

Reply:
Can you clarify where these two links point at?